kubernetes/adapter.go

```diff
@@ -120,6 +131,11 @@ func (a *Adapter) newIngressFromKube(kubeIngress *ingress) *Ingress {
 		ipAddressType = ipAddressTypeDualstack
 	}
 
+	sslPolicy := kubeIngress.getAnnotationsString(ingressSSLPolicyAnnotation, a.ingressDefaultSSLPolicy)
+	if _, ok := validSSLPolicy[sslPolicy]; !ok {
+		sslPolicy = a.ingressDefaultSSLPolicy
+	}
+
```

Comment:
This uses default policy if annotation is not set - that is expected. If annotation is set AND(BUT) policy is invalid it silently fallbacks to default if I read it correctly. Should it rather fail hard if it detects invalid policy?

Reply:
Problem of failing hard is we could break an existing ingress which is currently working for some clients, but then would break if misconfigured. We don't have a good signal to indicate that the ingress should continue to work in the previous configuration, but that the current configuration is broken.

I would like to work on better error reporting, but it's a bigger task which should be done separately.

I agree that if we merge this we can have hard to debug problems for users who misconfigure their ingress, but it's still better than the current state where the whole AWS account blows up.